libraries/rush-sdk/src/index.ts

```diff
@@ -21,7 +21,9 @@ import {
   sdkContext
 } from './helpers';
 
-const verboseEnabled: boolean = typeof process !== 'undefined' && process.env.RUSH_SDK_DEBUG === '1';
+const verboseEnabled: boolean =
+  typeof process !== 'undefined' &&
+  (process.env.RUSH_SDK_DEBUG === '1' || process.env._RUSH_SDK_DEBUG === '1');
```

Comment:
Why do we need a second env var?

Reply:
`RUSH_SDK_DEBUG` in disallowed in rush.js, which means it's impossible to run `RUSH_SDK_DEBUG=1 rush command-invoke-rush-sdk`.

A new environment variable with a prefix `_` make it possible to run `_RUSH_SDK_DEBUG=1 rush my-invoke-rush-sdk` to show debug loggings.